packages/vte.rb

```diff
@@ -43,10 +43,20 @@ class Vte < Meson
   gnome
   no_lto
 
-  meson_options '-D_systemd=false \
+  def self.build
+    system "meson setup #{CREW_MESON_FNO_LTO_OPTIONS} \
+      -D_systemd=false \
       -Dfribidi=true \
       -Dgtk3=true \
       -Dgtk4=true \
       -Dgir=false \
-      -Dvapi=false'
+      -Dvapi=false
+      builddir"
+    system 'meson configure builddir'
+    system "#{CREW_NINJA} -C builddir"
+  end
+
+  def self.install
+    system "DESTDIR=#{CREW_DEST_DIR} #{CREW_NINJA} -C builddir install"
+  end
```

Comment:
So we're abandoning the meson buildsystem to solve the problem?

Reply:
Yes, because otherwise, multiple packages (anything using `vte` can't be built.)

Reply:
Hopefully, #8541 will fix this and we can change later?